<launch>
	<!-- WARNING: MongoDB instance "mongodb_store" MUST be running-->
	<arg name="map"/> 	<!-- 2D Occupancy Grid Map -->
	<arg name="pointset"/>  <!--    Topological Map    -->
	
	<!-- lanch map server -->
	<node name="map_server" pkg="map_server" type="map_server" args="$(arg map)"/>
	<node name="static_transform_publisher" pkg="tf" type="static_transform_publisher" args="1 0 0 0 0 0 1 /map none 100"/>
	<node pkg="topological_navigation" type="map_manager.py" name="topological_map_manager" args="$(arg pointset)"/>
	<node pkg="topological_navigation" type="visualise_map.py" name="visualise_map" args="$(arg pointset) -e" respawn="true" output="screen"/> 
	<node pkg="tf" type="static_transform_publisher" name="map_tf_broadcaster" args="1 0 0 0 0 0 1 map world 100" />
	
	<node pkg="rviz" type="rviz" name="rviz" args="$(find topological_utils)/support/map_edition.rviz" /> 

</launch>
